### DISCUSSION TITLE
[ical.js] How should I name <package>-tests.ts when the package name is <name>.js?

### DISCUSSION THREAD

**Discussion by denik1981** [DefinitelyTyped/DefinitelyTyped#58440] [Request a new @types package] [answered]
I trying to build the types for the `ical.js` (not ical which a different project already here in @types).
I have my doubts on how should I properly name the tests file. Should the name be `ical.js-tests.ts`? 
Is that a valid name for the CI tests that will run later against these types?

I need a little help in here as it will be my first time writing @types for deftype.
The project repository is https://github.com/mozilla-comm/ical.js/

[a comment by an automated account is not shown]

**Comment by MysteryBlokHed** [comment, marked as answer]
The file with tests should be called `<package name>-tests.ts`, where the package name is exactly as it appears on npm. If you use dts-gen (DefinitelyTyped instructions [here](https://github.com/DefinitelyTyped/DefinitelyTyped#create-a-new-package)), the files will be created automatically.
In your case, the files could be made with `npx dts-gen --dt --name ical.js --template module`
